features/svg-opacity.yml

```diff
@@ -0,0 +1,10 @@
+name: SVG opacity
+description: The SVG `opacity` properties and attributes control the transparency of a stroke, fill, or entire SVG element.
```

Comment:
In general, this feature seems to run in the opposite direction of `paint-order` https://github.com/web-platform-dx/web-features/pull/1696/files#r1749729649. What makes this an SVG thing versus a CSS thing (like paint-order)? Not saying you're wrong here, but I am trying to learn the boundaries of CSS and SVG.

More description-specific stuff:

- If it's practical to do so (i.e., we're not talking about a dozen distinct names), name things in series rather than using wildcards. Sometimes we end up having to acknowledge a long list of things ("the Example API", "the example family of CSS properties", etc.) but if it's practical to include the actual characters developers would type, let's do it.
- Prefer to put the most specific thing first (i.e., "the example CSS property" not "the CSS example property"), to aid skimming. Imagine them all in a list and you'll probably see the issue: _everything_ ends up starting with the same-y "The SVG …" text.

```suggestion
description: The `opacity`, `fill-opacity`, and `stroke-opacity` SVG attributes control the transparency of a stroke, fill, or entire SVG element. The `fill-opacity` and `stroke-opacity` CSS properties correspond to the SVG attributes.
```

Reply:
> What makes this an SVG thing versus a CSS thing (like paint-order)?

Here's an attempt at documenting this-

- CSS properties that can be applied to HTML elements AND SVG elements (`font-family`, `writing-mode`) are generally grouped together. 
- CSS properties that can only be applied to SVG elements (`stop-color`, `vector-effect`) are generally a separate feature.

Some CSS properties for SVG elements also have analogous attributes, but this is orthogonal. The attributes should be follow the same rules, and be grouped with their related properties. 

Given this, I'm moving `svg.global_attributes.opacity` to the `opacity` feature.

Reply:
Thank you! It's very helpful to hear how you're thinking through this stuff.